modules/libjpeg_turbo/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://libjpeg-turbo.org/",
+    "maintainers": [],
```

Comment:
Would you be willing to add yourself as a maintainer?

Reply:
Sure, why not (though I'm not working on libjpeg-turbo, we use it as a dependency for JPEG-XL).
Will update PR soon.